docs/src/pages/components/table-generator.md

```diff
@@ -39,3 +39,22 @@ Este componente renderiza componentes dinamicamente através da prop `fields`, c
 Também é possível configurar o componente para executar uma ação ao clicar em uma linha, para isso, basta escutar o evento `row-click`.
 
 <doc-example file="QasTableGenerator/ClickableRow" title="Linha clicável" />
+
+Caso queira que ao clicar em uma linha, vá para outra rota, é possível utilizar a prop `row-route-fn` na qual o retorno deve ser um objeto com a estrutura aceita pelo vue-router.
```

Comment:
Elias só um detalhe, o `to` do vue router também aceita a string direta como `path` então não é somente um objeto.

Reply:
removido o detalhe de "um objeto"